src/storage/FileResourceStore.ts

```diff
@@ -15,10 +14,11 @@ import { NotFoundHttpError } from '../util/errors/NotFoundHttpError';
 import { UnsupportedMediaTypeHttpError } from '../util/errors/UnsupportedMediaTypeHttpError';
 import { InteractionController } from '../util/InteractionController';
 import { MetadataController } from '../util/MetadataController';
+import { ResourceMapper } from '../util/ResourceMapper';
 import { ensureTrailingSlash, trimTrailingSlashes } from '../util/Util';
 import { ResourceStore } from './ResourceStore';
 
-const { extname, join: joinPath, normalize: normalizePath } = posix;
+const { join: joinPath, normalize: normalizePath } = posix;
```

Comment:
Why `posix` and not directly `join` and `normalize` from `path` (which would be OS-independent)?

Reply:
Think this had something to do with Windows, @joachimvh probably knows why.

Reply:
```typescript
    return this.baseRequestURI + path.slice(this.rootFilepath.length);
```
gives a very wrong URL if the base path was previously composed using the normal join/normalize on windows instead of the posix one (you would have backslashes in the URL). This was the hotfix I added to make sure the unit tests of the file store still worked in windows. An even more correct solution would be to check what the `path.seperator` value is and take action based on whether that is a slash or not, but that's an issue for later.